ludwig/features/image_feature.py

```diff
@@ -294,32 +314,45 @@ def add_feature_data(
             # standard code anyway.
             if backend.supports_multiprocessing and (
                     num_processes > 1 or num_images > 1):
-                all_file_paths = [get_abs_path(src_path, file_path)
-                                  for file_path in input_df[feature[NAME]]]
+                all_img_stores = [get_abs_path(src_path, img_store)
```

Comment:
maybe we can rename `img_store` to `img_entry` (`img_entries` and `all_image_entries`) for consistency

Reply:
made change.